src/main/java/frc/robot/Robot.java

```diff
@@ -31,10 +31,10 @@
     public boolean usingVision = false;
     private String m_autoSelected;
     private final SendableChooser<String> m_chooser = new SendableChooser<>();
-    public Chassis chassis;
+    public static Chassis chassis;
```

Comment:
This seems like the only static field - why is this different from the others?

Reply:
fixed